rest-service/manager_rest/rest/resources_v3_1/deployments.py

```diff
@@ -130,13 +189,6 @@ def put(self, deployment_id, **kwargs):
                 raise self._error_from_create(deployment.create_execution)
         return deployment, 201
 
-    def _error_from_create(self, execution):
```

Comment:
Why was this moved nearer the start of the file?

Reply:
Well, just for formatting as all non resource methods aligned together

Reply:
Fair enough